cms/djangoapps/contentstore/tests/test_contentstore.py

```diff
@@ -1625,7 +1628,7 @@ def test_import_into_new_course_id(self):
         self.assertEqual(course_module.pdf_textbooks[0]["chapters"][1]["url"], '/static/Chapter2.pdf')
 
     def test_import_into_new_course_id_wiki_slug_renamespacing(self):
-        # If reimporting into the same course do not change the wiki_slug.
+        # If reimporting into the same course change the wiki_slug.
```

Comment:
I think it's incorrect to change this test case like this, please explain why it's happening